### PR TITLE
* packages/pfff/pfff.0.37.1/opam: new release

### PR DESCRIPTION
**Pull request by aryx**
The librairies in this package are needed for another project I want to submit soon to OPAM (efuns).

### CODE REVIEW

**Comment by aryx** [comment]
Is there a way to see the output of the failing command running for the Alpine distro. I can click on the logs but then the log just tells me there is an error in the configure command without any output I can look at.

File: packages/pfff/pfff.0.37.1/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
```
Comment (kit-ty-kate) [inline packages/pfff/pfff.0.37.1/opam]: This part is unnecessary. Opam 2 already takes care of removing files that have been installed. Could you remove it?

File: packages/pfff/pfff.0.37.1/opam [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+support for Erlang, Python, C#, Lisp, Scheme, Haskell, OPA, Sql, and
+even TeX. There is also very good support for OCaml and noweb
+(literate programming) so that pfff can be used on the code of pfff
+itself.
+
+For more information see https://github.com/facebook/pfff/wiki/Main"""
+extra-files: [
+]
+url {
+  src: "https://github.com/aryx/pfff/archive/0.37.1.tar.gz"
```
Comment (kit-ty-kate) [inline packages/pfff/pfff.0.37.1/opam]: Could you clarify the status of this? I see facebook stopped working on this a few years ago. Is it the new official repository or just your fork?

File: packages/pfff/pfff.0.37.1/opam [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+install: [
+  [make "install-libs"]
+]
+remove: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "uninstall-libs"]
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/pfff/pfff.0.37.1/opam]:
```suggestion
```
Where is this used? I don't even see use of ocamlfind or Findlib anywhere in the source code

File: packages/pfff/pfff.0.37.1/opam (hunk 2 above)
Comment (aryx) [reply packages/pfff/pfff.0.37.1/opam]: I am the main author of pfff and I left facebook in 2014. It is not maintained anymore by facebook
so I've forked it. I did not migrate yet the wiki to my fork, hence the mix of URLs.

File: packages/pfff/pfff.0.37.1/opam (hunk 3 above)
Comment (aryx) [reply packages/pfff/pfff.0.37.1/opam]: There is some occurences of ocamlfind in some Makefile, but for opam related stuff it's not used indeed. I've removed the dependency. Thx.

File: packages/pfff/pfff.0.37.1/opam (hunk 1 above)
Comment (aryx) [reply packages/pfff/pfff.0.37.1/opam]: sure

**Comment by aryx** [comment]
So how can I see the actual error with "Alipine failed" above. Clicling on Details and then Logs just show that configure returns an error without any output.

**Comment by mseri** [comment]
I don’t think there is a way from the web interface to get mor information than what is you have already seen

**Comment by aryx** [comment]
Can someone still accept my patch? I mean only Alpine users seem to be affected.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**